src/sidebar/components/MarkdownEditor.tsx

```diff
@@ -170,10 +169,11 @@ function ToolbarButton({
     );
   }
   return (
-    <IconButton
-      classes="px-2 py-2.5 text-tiny touch:text-base"
-      {...buttonProps}
-    />
+    <IconButton classes="px-2 py-2.5" {...buttonProps}>
+      {Icon && (
+        <Icon className="w-[10px] h-[10px] touch:w-[13px] touch:h-[13px]" />
```

Comment:
Once again using the `IconButton` pattern with a child `Icon` to directly apply sizing classes.